ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -390,179 +452,198 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
+                             this.setState({showInstitution: false}); }}
+                           checked={!this.state.showInstitution} style={{marginRight: '0.5rem'}}/>
+              <label style={{color: colors.primary}}>No</label>
             </div>
+          </Section>
+          {this.state.showInstitution &&
+          <FlexColumn style={{justifyContent: 'space-between'}}>
+            <TextInput data-test-id='institutionname' style={{width: '16rem', marginBottom: '0.5rem',
+              marginTop: '0.5rem'}}
+              value={this.getInstitutionalAffiliationPropertyOrEmptyString('institution')}
+              placeholder='Institution Name'
+              onChange={value => this.updateInstitutionAffiliation('institution', value)}
+                       />
+            <Dropdown data-test-id='institutionRole'
+                      value={this.getInstitutionalAffiliationPropertyOrEmptyString('role')}
+                      onChange={e => this.updateInstitutionAffiliation('role', e.value)}
+                      placeholder='Which of the following describes your role'
+                      style={{width: '16rem'}} options={AccountCreationOptions.roles}/>
+          </FlexColumn>}
+          {!this.state.showInstitution &&
+          <FlexColumn style={{justifyContent: 'space-between'}}>
+            <Dropdown data-test-id='affiliation'
+                      style={{width: '18rem', marginBottom: '0.5rem', marginTop: '0.5rem'}}
+                      value={this.getInstitutionalAffiliationPropertyOrEmptyString('nonAcademicAffiliation')}
+                      options={AccountCreationOptions.nonAcademicAffiliations}
+                      onChange={e => this.updateNonAcademicAffiliationRoles(e.value)}
+                      placeholder='Which of the following better describes your affiliation?'/>
+            {this.state.showNonAcademicAffiliationRole &&
+            <Dropdown data-test-id='affiliationrole' placeholder='Which of the following describes your role'
+                      options={this.state.rolesOptions}
+                      value={this.getInstitutionalAffiliationPropertyOrEmptyString('role')}
+                      onChange={e => this.selectNonAcademicAffiliationRoles(e.value)}
+                      style={{width: '18rem'}}/>}
+            {this.state.showNonAcademicAffiliationOther &&
+            <TextInput value={this.getInstitutionalAffiliationPropertyOrEmptyString('other')}
+                       onChange={value => this.updateInstitutionAffiliation('other', value)}
+                       style={{marginTop: '1rem', width: '18rem'}}/>}
+          </FlexColumn>}
+          <FormSection style={{paddingBottom: '1rem'}}>
+            <Button disabled={this.state.usernameCheckInProgress || this.isUsernameValidationError}
+                    style={{'height': '2rem', 'width': '10rem'}}
+                    onClick={() => this.validateAccountCreation()}>
```

Comment:
While trying this on my local i cannot see any required field error on Clicking the next button

Reply:
Yes, I haven't super handled validation yet. I am planning on following up on that, but there should be no tooltip yet. I think the error shows up at the top of the page right now?

Reply:
I do not see any errors on the top of my page as well

Reply:
Ah, yeah, I don't know if there's required field validation yet, thinking about it. I want to add that with this https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&modal=detail&selectedIssue=RW-4283 for a full validation overhaul.